src/pages/login/index.tsx

```diff
@@ -84,15 +87,23 @@ function LoginPage() {
                 className="!mb-1 mt-3"
                 onChange={handleChange('email')}
               />
-              <Input
-                type="password"
-                value={values.password}
-                error={touched.password ? errors.password : undefined}
-                required
-                placeholder={t('login:password')}
-                className="!my-1"
-                onChange={handleChange('password')}
-              />
+              <div className="relative">
+                <Input
+                  type={showPassword ? 'text' : 'password'}
+                  value={values.password}
+                  error={touched.password ? errors.password : undefined}
+                  required
+                  placeholder={t('login:password')}
+                  className="!my-1"
+                  onChange={handleChange('password')}
+                />
+                <span
+                  className="absolute right-3 top-2 cursor-pointer"
+                  onClick={() => setShowPassword(!showPassword)}
+                >
+                  {showPassword ? '\u{1F440}' : '\u{1F648}'}
```

Comment:
Do not use native emoji. Please use eye and eye-slash components from heroicons library